draft-irtf-cfrg-hpke.md

```diff
@@ -296,6 +299,89 @@ plaintexts.
 The procedures described in this session are laid out in a
 Python-like pseudocode.  The ciphersuite in use is left implicit.
 
+## Key Schedule
+
+The variants of HPKE defined in this document share a common
+mechanism for translating the inputs to the protocol into an
+encryption context.  The inputs to the key schedule are as follows:
+
+* `pkR` - The receiver's public key
+* `zz` - A shared secret generated via the KEM for this transaction
+* `enc` - An encapsulated key produced by the KEM for the receiver
+* `info` (optional) - Application-supplied information
+* `psk` (optional) - A pre-shared secret held by both the initiator
+  and the receiver.
+* `pskID` (optional) - An identifier for the PSK
+* `pkI` (optional) - The initiator's public key
+
+If any of the optional values is not present (e.g., set to `None`
+in a Python implementation), then it is treated in the same way as
+an empty octet string.
+
+The key and nonce produced by this algorithm have the property that
+they are only known to the holder of the receipient private key, and
```

Comment:
This seems to suggest that the key and nonce are outputs, but they're just put in the context. Perhaps we replace "produced" with "computed"?